### DISCUSSION TITLE
Stamen x Stadia - reclineview  - Maps view

### DISCUSSION THREAD

**Discussion by carlosmartinezdelacasa** [ckan/ckan#7875] [Q&A]
After October 2023, redirects may stop working and those still pointing to stamen.com or fastly.com URLs will see a blank map.

I understand that this affects the reclineview libraries since the Maps view associated with these calls will stop working.

Is there an option planned, another open source library...

As solved in old versions of CKAN 2.9.X.

**Comment by muhammetcetinyamac** [comment]
Hello, yes, I'm curious about this topic as well. Do you have any suggestions for a map infrastructure for both the current version and previous versions of CKAN?

**Comment by muhammetcetinyamac** [comment]
Hello, as a solution, I replaced the base map layer with the open-source OSM infrastructure using the latest version of ckanext-geoview. Currently, the map can be displayed. In the future, if there are changes to the base map, I will continue to follow updates and make necessary adjustments.
